graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/ClusterAdapterES7.java

```diff
@@ -104,7 +104,14 @@ public Set<NodeFileDescriptorStats> fileDescriptorStats() {
     }
 
     private List<NodeResponse> nodes() {
-        return catApi.nodes();
+        final List<NodeResponse> allNodes = catApi.nodes();
+        final List<NodeResponse> nodesWithDiskStatistics = allNodes.stream().filter(NodeResponse::hasDiskStatistics).collect(Collectors.toList());
+        if (allNodes.size() != nodesWithDiskStatistics.size()) {
+            final List<NodeResponse> nodesWithMissingDiskStatistics = allNodes.stream().filter(nr -> !nr.hasDiskStatistics()).collect(Collectors.toList());
+            LOG.info("_cat/nodes API has returned " + nodesWithMissingDiskStatistics.size() + " nodes without disk statistics:");
```

Comment:
What about a WARNING level instead of info?

Reply:
To be honest, I am not sure if this situation is normal or natural. We have one client reporting it, we do not know the details about why it is so. I am a little bit hesitant to use too high logging level, as I am not quite sure that it is smth user needs to be very concerned about...

Reply:
Then let's keep it to info and adapt afterwards if needed :+1: